utils/fetchEarthquakes.ts

```diff
@@ -70,6 +70,34 @@ export const fetchDailyGraphStats = async (): Promise<any> => {
 };
 
 export const fetchWeeklyStats = async (): Promise<any> => {
+  const res = await axios.get(
+    'https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/all_week.geojson'
+  );
+
+  const weeklyEvents = res.data.features;
+
+  // sort events by date ensure days are in correct order, ending with today
+  const eventsByDate = weeklyEvents.reduce(
+    (acc: { [date: string]: number }, feature: EarthquakeFeature) => {
+      const date = new Date(feature.properties?.time)
+        .toISOString()
+        .split('T')[0];
+      if (!acc[date]) {
+        acc[date] = 0;
+      }
+      acc[date]++;
+      return acc;
+    },
+    {} as { [date: string]: number }
+  );
+
+  return {
+    weeklyEvents,
+    eventsByDate,
+  };
+};
+
+export const fetchWeeklyGraphStats = async (): Promise<any> => {
```

Comment:
Added for the same reason we added the "daily" version of this function - https://github.com/ash-bergs/earthquake-tracker/issues/28 

This will fall into that ticket 👆